packages/nx/src/nx-cloud/utilities/url-shorten.ts

```diff
@@ -10,9 +10,10 @@ export async function createNxCloudOnboardingURL(
   accessToken?: string,
   meta?: string,
   forceManual = false,
-  forceGithub = false
+  forceGithub = false,
+  directory?: string
```

Comment:
This is needed during CNW, since the `process.cwd()` is not where the Nx workspace is. During `nx connect` or `nx init` it was fine but CNW never worked to find git remote info.